backend/api.test/EndpointTest.cs

```diff
@@ -144,7 +145,8 @@ private async Task<(string installationId, string plantId, string deckId, string
             {
                 InstallationCode = installationCode,
                 PlantCode = plantCode,
-                Name = deckName
+                Name = deckName,
+                LocalizationPose = testPose
```

Comment:
Area has DefaultLocalizationPose and deck has LocalizationPose. Consider renaming one of them for consistent naming